formats/financial_releases_index/publisher/details.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+   "required": [
+     "title",
+     "image",
+     "press_releases"
+   ],
+  "properties": {
+     "title": {
+       "type": "string"
+     },
+    "image": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "url": {
+          "type": "string",
+          "format": "uri"
+        },
+        "alt_text": {
+          "type": "string"
+        }
+      }
+    },
+    "press_releases": {
```

Comment:
These should be in the links hash rather than embedded within details, linking to the `financial_press_release` content items.